src/css/angular-tooltips.css

```diff
@@ -36,44 +36,87 @@
   transition: opacity 0.2s linear;
   opacity: 1;
 }
+._720kb-tooltip-caret:before {
```

Comment:
That's my fault. 
I add this because in our design, I need to set a border for the whole tooltip including the triangle. So, I use two triangle: the "before" one with larger size works as the border, the "after" one with smaller size works as the content. Just like what github does. You can check the style of github.
![image](https://cloud.githubusercontent.com/assets/2586026/5787439/caff9046-9e39-11e4-9632-fabab4e0b270.png)